yarn-project/ethereum/src/l1_tx_utils.test.ts

```diff
@@ -0,0 +1,189 @@
+import { EthAddress } from '@aztec/foundation/eth-address';
+import { createDebugLogger } from '@aztec/foundation/log';
+
+import { type Anvil, createAnvil } from '@viem/anvil';
+import getPort from 'get-port';
+import { createPublicClient, createWalletClient, http } from 'viem';
+import { mnemonicToAccount, privateKeyToAccount } from 'viem/accounts';
+import { foundry } from 'viem/chains';
+
+import { L1TxUtils } from './l1_tx_utils.js';
+
+const MNEMONIC = 'test test test test test test test test test test test junk';
+const WEI_CONST = 1_000_000_000n;
+// Simple contract that just returns 42
+const SIMPLE_CONTRACT_BYTECODE = '0x69602a60005260206000f3600052600a6016f3';
+
+const startAnvil = async (l1BlockTime?: number) => {
+  const ethereumHostPort = await getPort();
+  const rpcUrl = `http://127.0.0.1:${ethereumHostPort}`;
+  const anvil = createAnvil({
+    port: ethereumHostPort,
+    blockTime: l1BlockTime,
+  });
+  await anvil.start();
+  return { anvil, rpcUrl };
+};
+
+describe('GasUtils', () => {
+  let gasUtils: L1TxUtils;
+  let publicClient: any;
+  let walletClient: any;
```

Comment:
```suggestion
  let walletClient: WalletClient<HttpTransport, Chain, Account>,
  let publicClient: PublicClient<HttpTransport, Chain>,
```
Viem can be super annoying with types.